public/app/features/connections/components/ConnectionsRedirectNotice/ConnectionsRedirectNotice.tsx

```diff
@@ -0,0 +1,54 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { Alert, LinkButton, useStyles2 } from '@grafana/ui';
+
+import { ROUTES } from '../../constants';
+
+const getStyles = (theme: GrafanaTheme2) => ({
+  alert: css`
+  > div:nth-child(2) {
```

Comment:
Just wondering whether we want to rely on the internal implementation of the Alert component, what if we keep the text as title and set something like .alertButton {position: absolute; right: 10px} ?

Reply:
I fixed this, but then we're pretty much unable to make it look like it's designed. We can still get away with it, I think.

Reply:
I think you can add text as a title and position the button absolutely to make it look like before?

Reply:
I'm not sure, but that also feels like a hack to me :/